background/providers/rss/tests/testdata.js

```diff
@@ -0,0 +1,172 @@
+        description: '',
+        seeds: 0,
+        peers: 0,
+        pubdate: new Date().toString('yyyyMMddHHmmss'),
+        enclosure: {
+            url: 'http://releases.ubuntu.com/16.04/ubuntu-16.04.2-desktop-i386.iso.torrent',
+            length: 0,
+            type: 'application/x-bittorrent'
+        }
+    },
+    {
+        title: 'Ubuntu 16.04.2 Server (64-bit)',
+        category: 'Other',
+        contentLength: 0,
+        description: '',
+        seeds: 0,
+        peers: 0,
+        pubdate: new Date().toString('yyyyMMddHHmmss'),
+        enclosure: {
+            url: 'http://releases.ubuntu.com/16.04/ubuntu-16.04.2-server-amd64.iso.torrent',
+            length: 0,
+            type: 'application/x-bittorrent'
+        }
+    },
+    {
+        title: 'Ubuntu 16.04.2 Server (32-bit)',
+        category: 'Other',
+        contentLength: 0,
+        description: '',
+        seeds: 0,
+        peers: 0,
+        pubdate: new Date().toString('yyyyMMddHHmmss'),
+        enclosure: {
+            url: 'http://releases.ubuntu.com/16.04/ubuntu-16.04.2-server-i386.iso.torrent',
+            length: 0,
+            type: 'application/x-bittorrent'
+        }
+    },
+    {
+        title: 'Ubuntu 14.04.5 Desktop (64-bit)',
+        category: 'Other',
+        contentLength: 0,
+        description: '',
+        seeds: 0,
+        peers: 0,
+        pubdate: new Date().toString('yyyyMMddHHmmss'),
+        enclosure: {
+            url: 'http://releases.ubuntu.com/14.04/ubuntu-14.04.5-desktop-amd64.iso.torrent',
+            length: 0,
+            type: 'application/x-bittorrent'
+        }
+    },
+    {
+        title: 'Ubuntu 14.04.5 Desktop (32-bit)',
+        category: 'Other',
+        contentLength: 0,
+        description: '',
+        seeds: 0,
+        peers: 0,
+        pubdate: new Date().toString('yyyyMMddHHmmss'),
+        enclosure: {
+            url: 'http://releases.ubuntu.com/14.04/ubuntu-14.04.5-desktop-i386.iso.torrent',
+            length: 0,
+            type: 'application/x-bittorrent'
+        }
+    },
+    {
+        title: 'Ubuntu 14.04.5 Server (64-bit)',
+        category: 'Other',
+        contentLength: 0,
+        description: '',
+        seeds: 0,
+        peers: 0,
+        pubdate: new Date().toString('yyyyMMddHHmmss'),
+        enclosure: {
+            url: 'http://releases.ubuntu.com/14.04/ubuntu-14.04.5-server-amd64.iso.torrent',
+            length: 0,
+            type: 'application/x-bittorrent'
+        }
+    },
+    {
+        title: 'Ubuntu 14.04.5 Server (32-bit)',
+        category: 'Other',
+        contentLength: 0,
+        description: '',
+        seeds: 0,
+        peers: 0,
+        pubdate: new Date().toString('yyyyMMddHHmmss'),
+        enclosure: {
+            url: 'http://releases.ubuntu.com/14.04/ubuntu-14.04.5-server-i386.iso.torrent',
+            length: 0,
+            type: 'application/x-bittorrent'
+        }
+    }
+];
+
+export {torrents as default};
```

Comment:
Default exports should be in the style of `export default torrents;`.

If you need to export more than a single item please use this format.

```
export {
    torrents
};
```